packages/craco/lib/features/webpack/style/css.js

```diff
@@ -1,15 +1,29 @@
 const { getLoaders, loaderByName } = require("../../../loaders");
 const { log, logError } = require("../../../logger");
-const { isFunction, deepMergeWithArray } = require("../../../utils");
+const { isFunction, isBoolean, deepMergeWithArray } = require("../../../utils");
 
 function setModuleLocalIdentName(match, localIdentName) {
-    if (match.loader.options) {
-        delete match.loader.options.getLocalIdent;
-        match.loader.options.localIdentName = localIdentName;
+    // The css-loader version of create-react-app has changed from 2.1.1 to 3.2.0
+    // https://github.com/facebook/create-react-app/commit/f79f30
+    if (isBoolean(match.loader.options.modules)) {
+        if (match.loader.options) {
+            delete match.loader.options.getLocalIdent;
+            match.loader.options.localIdentName = localIdentName;
+        } else {
+            match.loader.options = {
+                localIdentName: localIdentName
+            };
+        }
     } else {
-        match.loader.options = {
-            localIdentName: localIdentName
-        };
+        // This setting applies to create-react-app@3.3.0
+        if (match.loader.options) {
+            delete match.loader.options.modules.getLocalIdent;
+            match.loader.options.modules.localIdentName = localIdentName;
+        } else {
+            match.loader.options.modules = {
```

Comment:
Will not work since if you get there it means `match.loader.options` is undefined

Reply:
Since there is already a check for `match.loader.options` there

```js
const cssModuleLoaders = matches.filter(x => x.loader.options && x.loader.options.modules);
```
I guess the check for `options` and `modules` could be removed from this function ,what do you think?

Reply:
Thank you for the feedback!

I was checking in the previous code, so I thought I should check.

But I think your feedback is correct.

Reply:
Yes sorry the original code wasn't right.